src/core/data/weapon_tints.hpp

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+namespace big
+{
+	constexpr const static auto default_tints = std::to_array({
+		"Black tint",
+		"Green tint",
+		"Gold tint",
+		"Pink tint",
+		"Army tint",
+		"LSPD tint",
+		"Orange tint",
+		"Platinum tint"
+	});
+
+	constexpr const static auto mk2_tints = std::to_array({
+		"Classic Black",
+		"Classic Grey",
+		"Classic Two - Tone",
+		"Classic White",
+		"Classic Beige",
+		"Classic Green",
+		"Classic Blue",
+		"Classic Earth",
+		"Classic Brown & Black",
+		"Red Contrast",
+		"Blue Contrast",
+		"Yellow Contrast",
+		"Orange Contrast",
+		"Bold Pink",
+		"Bold Purple & Yellow",
+		"Bold Orange",
+		"Bold Green & Purple",
+		"Bold Red Features",
+		"Bold Green Features",
+		"Bold Cyan Features",
+		"Bold Yellow Features",
+		"Bold Red & White",
+		"Bold Blue & White",
+		"Metallic Gold",
+		"Metallic Platinum",
+		"Metallic Grey & Lilac",
+		"Metallic Purple & Lime",
+		"Metallic Red",
+		"Metallic Green",
+		"Metallic Blue",
+		"Metallic White & Aqua",
+		"Metallic Red & Yellow"
+	});

```

Comment:
Now that we have access to the script functions, can't we grab these at runtime so they're translated?

Reply:
Unfortunately, there isn't a suitable function we can use for this. Instead, I've modified the code to generate the translations using `GET_FILENAME_FOR_AUDIO_CONVERSATION` when the user opens the tab.